motion_web/pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: motion_web
 description: The web implementation for the Motion plugin.
-version: 1.4.0
+version: 1.4.1
 homepage: https://cendre.me
 repository: https://github.com/mrcendre/motion
 issue_tracker: https://github.com/mrcendre/motion/issues?q=is%3Aissue+is%3Aopen
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.20.0"
+  sdk: ">=3.3.0"
+  flutter: ">=3.19.0"
```

Comment:
This change is actually a big deal, but I think it is acceptable because Flutter 3 has been around for long enough now. It will require to bump all packages to a new major version though, as these changes might be breaking.

I will take care of it as soon as #16 is merged.